documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -69,30 +108,81 @@ value.converter.apicurio.registry.url=http://apicurio:8080/api
+Internally, Kafka Connect always uses JSON key/value converters for storing configuration and offsets.
+
+// Type: procedure
+// Title: Deploying connectors that use Avro in {prodname} containers
+// ModuleID: deploying-connectors-that-use-avro-in-debezium-containers
+== Deploying with {prodname} containers
+
+In your environment, you might want to use a {prodname} pre-built container to deploy a {prodname} connector that uses Avro serializaion. Follow the procedure here to do that. In this procedure, you build a {prodname} image in which {prodname} uses the Avro converter.  
+
+.Prerequisites
 
-== {prodname} Container Images
+* You have the required permissions on a Kafka cluster.
+* You downloaded the {prodname} connector plug-in that you want to deploy with Avro serialization.
 
-Deploy an Apicurio Registry instance (this example uses a non-production in-memory instance):
+.Procedure
 
-[source,subs="+attributes"]
+. Deploy an instance of {registry}. 
++
+ifdef::community[]
+The following example uses a non-production, in-memory, {registry}  instance:
++
+[source,subs="attributes"]
 ----
 docker run -it --rm --name apicurio \
     -p 8080:8080 apicurio/apicurio-registry-mem:{apicurio-version}
 ----
+endif::community[]
+ifdef::product[]
+.. Choose the OpenShift project in which you want to deploy the {prodname} connector. In the following command, `$NAMESPACE` represents your project.  
+.. Deploy the latest published {registry} operator by running the following command:
++
+`curl -sSL https://raw.githubusercontent.com/apicurio/apicurio-registry-operator/master/docs/resources/install.yaml | sed "s/{NAMESPACE}/$NAMESPACE/g" | oc apply -f -`
++
+This deploys the latest development version of the {registry} operator from the `master` branch. To deploy other versions, specify a different branch or tag, or edit the operator image reference in the file.
++
+.. Create a new {registry} deployment by specifying the in-memory persistence option in one of the example custom resources, for example:
++
+`oc create -f https://raw.githubusercontent.com/apicurio/apicurio-registry-operator/master/docs/resources/example-cr/in-memory.yaml`
++
+The in-memory deployment is not suitable for production. Use the Kafka Streams persistence option for production. For more information, see {LinkServiceRegistryGetStart}[NameServiceRegistryGetStart].
+endif::product[]
+
+. Build a {prodname} container image that implements the Avro converter:
++
+ifdef::community[]
+.. Copy link:https://github.com/debezium/debezium-examples/blob/master/tutorial/debezium-with-apicurio/Dockerfile[`Dockerfile`] to a convenient location. This file has the following content: 
++
+[listing,options="nowrap"]
+----
+ARG DEBEZIUM_VERSION
+FROM debezium/connect:$DEBEZIUM_VERSION
+ENV KAFKA_CONNECT_DEBEZIUM_DIR=$KAFKA_CONNECT_PLUGINS_DIR/debezium-connector-mysql
+ENV APICURIO_VERSION=1.1.2.Final
```

Comment:
There's a variable defined for the version (in antora.yml in case of upstream docs): {apicurio-version}. Enable substitution for the listing in order to reference it.

Reply:
Done. In next commit.